gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/gc/GcProfile.java

```diff
@@ -46,7 +47,18 @@
         private int initiatingOccupancyFraction = 68;
 
         @Override
-        public final List<String> gcJvmOpts() {
+        public final List<String> gcJvmOpts(JavaVersion javaVersion) {
+            // TODO(forozco): Update to use proper constant once we can support gradle 6.0
+            if (javaVersion == JavaVersion.VERSION_HIGHER) {
+                return ImmutableList.of(
+                        "-XX:+UnlockExperimentalVMOptions",
+                        // https://wiki.openjdk.java.net/display/shenandoah/Main
+                        "-XX:+UseShenandoahGC",
+                        // "forces concurrent cycle instead of Full GC on System.gc()"
+                        "-XX:+ExplicitGCInvokesConcurrent",
+                        "-XX:+ClassUnloadingWithConcurrentMark",
+                        "-XX+UseNUMA");
```

Comment:
this is on by default https://github.com/openjdk/jdk/blob/master/src/hotspot/share/gc/shenandoah/shenandoahArguments.cpp#L62

Reply:
I'd prefer to explicitly set it given that the [shenandoah](https://wiki.openjdk.java.net/display/shenandoah/Main#Main-Basicconfiguration) docs encourage setting the flag and not having the flag set results in a performance loss